string zone

string name
string desc
string code
duration timeout
string team
uint8 round
uint8 run
time schedule

duration timer
string state

string manual_operation

bool connect_enabled
bool disconnect_enabled
bool start_enabled
bool stop_enabled
bool prev_enabled
bool next_enabled

bool omf
uint8[] omf_switches
uint8 omf_damaged
bool omf_complete

string log
string online_data

ZoneScoreGroup[] scoring
